src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -860,13 +856,13 @@ private async Task HandleConfigurationPropertiesAsync(ConnectionInfo sshConnecti
             await HandleConfigurationPropertyAsync(sshConnectionInfo, "KeepSshPortOpen", configuration.KeepSshPortOpen, "env-12-keep-ssh-port-open.txt");
         }
 
-        private async Task HandleConfigurationPropertyAsync(ConnectionInfo sshConnectionInfo, string key, string value, string envFileName)
+        private static async Task HandleConfigurationPropertyAsync(ConnectionInfo sshConnectionInfo, string key, string value, string envFileName)
         {
             // If the value is provided and empty, remove the property from the VM
             // If the value is not empty, create/update the property on the VM
             // If the value is not provided, don't do anything, the property may or may not exist on the VM
             // Properties are kept in env-* files, aggregated to .env file at VM startup, and used in docker-compose.yml as environment variables
-            if (value != null && value.Equals(string.Empty))
+            if (value is not null && value.Equals(string.Empty))
```

Comment:
equivalent of this?

if (value?.Equals(string.Empty))

Reply:
Actually, yes, it is. I'll catch it on the next round.